test/tests.bats

```
@@ -10,7 +10,7 @@ setup() {
 }
 
 setup_file() {
-  PRIVATE_CONFIG="$(duplicate_config_for_container . mail)"
+  local PRIVATE_CONFIG="$(duplicate_config_for_container . mail)"
```

Again local hides failures in `$()`

---

Done.